docs/source/api/python/dgl.dataloading.rst

```diff
@@ -48,3 +48,17 @@ to generate negative edges.
 
 .. autoclass:: Uniform
     :members: __call__
+
+Async Copying to/from GPUs
+--------------------------
+.. currentmodule:: dgl.dataloading
+
+Data can be copied from the CPU to the GPU, or from the GPU to the CPU,
+while the GPU is being used for
+computation, using the :class:`AsyncTransferer`.
+
+.. autoclass:: AsyncTransferer
+    :members: __init__, async_copy
+
+.. autoclass:: async_transferer.Transfer
+    :members: wait
```

Comment:
should these APIs exposed to users?

Reply:
generally I feel people are reading all the public codes so there is hardly a line between internal and external APIs. I would well-doc it assuming people will use it.

Reply:
once it is documented, i would consider that we officially announce it as part of DGL API. Then it's our responsibility to make the API stable.